compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/lower/loops/ProgressionHandlers.kt

```diff
@@ -375,16 +380,22 @@ internal class ArrayIterationHandler(context: CommonBackendContext) : IndexedGet
         get() = getClass()!!.properties.first { it.name == Name.identifier("size") }.getter!!
 
     override val IrType.getFunction
-        get() = getClass()!!.functions.first { it.name.asString() == "get" }
+        get() = getClass()!!.functions.first { it.name == Name.identifier("get") }
```

Comment:
I would prefer to use 'name.asString() == "get"' instead of Name object creation or at least extract Name object to global properties

Reply:
Done, thanks!

Reply:
@madsager Note `asString`, not `toString` :) There's no difference right now, but we don't usually call `toString` in production code

Reply:
d'oh! ;-)

Done!